From 35d1532197f26ee0c1ae3cf73cecc76b700d8574 Mon Sep 17 00:00:00 2001
From: Markus Armbruster <armbru@redhat.com>
Date: Tue, 18 May 2010 19:31:39 -0300
Subject: [PATCH 03/20] doc: Fix acl monitor command documentation

RH-Author: Markus Armbruster <armbru@redhat.com>
Message-id: <1274211101-5748-3-git-send-email-armbru@redhat.com>
Patchwork-id: 9378
O-Subject: [RHEL-6 KVM PATCH 2/4] doc: Fix acl monitor command documentation
Bugzilla: 588719
RH-Acked-by: Gerd Hoffmann <kraxel@redhat.com>
RH-Acked-by: Kevin Wolf <kwolf@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>

Commit 15dfcd45 added acl_add and acl_reset, but fat-fingered their
documentation to read acl_allow and acl_remove.

Signed-off-by: Markus Armbruster <armbru@redhat.com>
Signed-off-by: Anthony Liguori <aliguori@us.ibm.com>
(cherry picked from commit 0e4aec988c813227dccff705c2cd10b0f954bf23)
---
 qemu-monitor.hx |    7 ++++---
 1 files changed, 4 insertions(+), 3 deletions(-)

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 qemu-monitor.hx |    7 ++++---
 1 files changed, 4 insertions(+), 3 deletions(-)

diff --git a/qemu-monitor.hx b/qemu-monitor.hx
index b8727c1..c8d07a4 100644
--- a/qemu-monitor.hx
+++ b/qemu-monitor.hx
@@ -1061,8 +1061,8 @@ ETEXI
     },
 
 STEXI
-@item acl_allow @var{aclname} @var{match} @code{allow|deny} [@var{index}]
-@findex acl_allow
+@item acl_add @var{aclname} @var{match} @code{allow|deny} [@var{index}]
+@findex acl_add
 Add a match rule to the access control list, allowing or denying access.
 The match will normally be an exact username or x509 distinguished name,
 but can optionally include wildcard globs. eg @code{*@@EXAMPLE.COM} to
@@ -1094,7 +1094,8 @@ ETEXI
     },
 
 STEXI
-@item acl_remove @var{aclname}
+@item acl_reset @var{aclname}
+@findex acl_reset
 Remove all matches from the access control list, and set the default
 policy back to @code{deny}.
 ETEXI
-- 
1.7.0.3

